src/SourceGenerators.BootstrapDrivers/AnalyzerReleases.Unshipped.md

```
@@ -0,0 +1,8 @@
+; Unshipped analyzer release
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)